pkg/liqoctl/install/aks/provider.go

```diff
@@ -161,6 +172,11 @@ func (k *aksProvider) parseClusterOutput(ctx context.Context, cluster *container
 
 	k.endpoint = *cluster.Fqdn
 
+	if cluster.Location != nil {
+		k.clusterLabels[consts.TopologyRegionClusterLabel] = *cluster.Location
+		k.clusterLabels[consts.TopologyZoneClusterLabel] = *cluster.Location
```

Comment:
It seems to me region and zone are always assigned the same value. Is it worth to keep both?